Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -239,15 +238,15 @@ - (DocumentViewChange)parseChange:(NSDictionary *)jsonDoc ofType:(DocumentViewCh
 #pragma mark - Methods for doing the steps of the spec test.
 
 - (void)doListen:(NSArray *)listenSpec {
-  FSTQuery *query = [self parseQuery:listenSpec[1]];
+  Query query = [self parseQuery:listenSpec[1]];
   TargetId actualID = [self.driver addUserListenerWithQuery:query];
 
   TargetId expectedID = [listenSpec[0] intValue];
   XCTAssertEqual(actualID, expectedID, @"targetID assigned to listen");
 }
 
 - (void)doUnlisten:(NSArray *)unlistenSpec {
-  FSTQuery *query = [self parseQuery:unlistenSpec[1]];
+  Query query = [self parseQuery:unlistenSpec[1]];
```

Comment:
std::move? Also immediately above.

Also, I'm pretty sure I'm missing plenty of these; I'd look for more, but every time I attempt to expand the context, github "helpfully" scrolls me to somewhere random in the PR, forcing me to figure out where I was and go back. It's nuts. So I've given up, at least for the test files. (A couple extra copies in the tests aren't going to hurt us.)

Reply:
Done here and above.

I took a pass through all instances matching `\bQuery\b\s+\w+\s*=` and adjusted as necessary.